freqtrade/freqai/freqai_interface.py

```diff
@@ -198,9 +199,16 @@ def shutdown(self):
         self.data_provider = None
         self._on_stop()
 
-        logger.info("Waiting on Training iteration")
-        for _thread in self._threads:
-            _thread.join()
+        if self.freqai_info.get("wait_for_training_iteration_on_reload", True):
```

Comment:
I guess we'll want to add it to the docs?

otherwise i doubt anyone will know how to set this.

Reply:
maybe as an addition to the above comment - it should probably also be added to `config_schema.py` - so it's visible in "editor completion" when the config is setup properly.